src/Files.App/Dialogs/FilesystemOperationDialog.xaml

```diff
@@ -134,7 +111,7 @@
 					Width="200"
 					HorizontalAlignment="Right"
 					VerticalAlignment="Center"
-					x:Load="{x:Bind IsConflict, Mode=OneWay}"
+					Visibility="{x:Bind IsConflict, Mode=OneWay, Converter={StaticResource BoolToVisibilityConverter}}"
```

Comment:
x:Load doesn't work?

Reply:
Yes, it seems that the combo box is always displayed with the default values with x:Load.

Reply:
Can you add a comment to the xaml so future me doesn't go and break this?

Reply:
Ok, added comment.